osmosis-1/osmosis.zone_assets.json

```diff
@@ -2286,6 +2286,12 @@
       "override_properties": {
         "traces": []
       }
+    },
+    {
+      "chain_name": "pundix",
+      "base_denom": "bsc0x29a63F4B209C29B4DC47f06FFA896F32667DAD2C",
+      "path": "transfer/channel-12618/bsc0x29a63F4B209C29B4DC47f06FFA896F32667DAD2C",
+      "osmosis_verified": false
     }
```

Comment:
please add "osmosis_unlisted": "true".
We will revert this once we've completed validation.